src/config.rs

```diff
@@ -304,6 +316,32 @@ impl Context {
                 let value = value.map(improve_single_line_input);
                 self.sql.set_raw_config(key, value.as_deref()).await?;
             }
+            Config::DebugLogging => {
+                if value == Some("0") || value == Some("") || value == None {
+                    if let Some(webxdc_message_id) =
+                        self.sql.get_raw_config_u32(Config::DebugLogging).await?
+                    {
+                        message::delete_msgs(self, &[MsgId::new(webxdc_message_id)]).await?;
+                    }
+                    self.sql.set_raw_config(key, value).await?;
+                    self.debug_logging.store(0, atomic::Ordering::Relaxed);
+                } else {
+                    let data: &[u8] = include_bytes!("../assets/webxdc_logging.xdc");
```

Comment:
The `webxdc_logging.xdc` file is minified; the source at https://github.com/Septias/webxdc_logging should probably be put into this repo here (alternatively, a link will also do). Before, some things will have to be adapted, e.g. `create-xdc.sh` currently moves the created file to `/home/septias/Downloads/` and `.git` and `.github` have to be deleted, maybe also `.vscode`.

But I'm probably missing some things that have to be done, so @Septias @Simon-Laux could one of you do this?

Reply:
I will take a look later